packages/devtools/src/formats/runtime.ts

```diff
@@ -344,14 +344,21 @@ export function createRuntimeDefinition(
         runtime.accountData[key + 'List'] = { type: 'connection', name: modelName }
       } else {
         const relationType = modelDefinition.accountRelation.type
-        if (relationType === 'single') {
-          runtime.accountData[key] = { type: 'node', name: modelName }
-          // @ts-ignore TS2367, should be unnecessary check based on type definition but more types
-          // could be added later
-        } else if (relationType === 'list') {
-          runtime.accountData[key + 'List'] = { type: 'connection', name: modelName }
-        } else {
-          throw new Error(`Unsupported account relation type: ${relationType as string}`)
+        switch (relationType) {
+          case 'list':
+            runtime.accountData[key + 'List'] = { type: 'connection', name: modelName }
+            break
+          case 'set':
+            // relation to single document in the set based on input
+            runtime.accountData[key] = { type: 'set', name: modelName }
+            // relation to all documents in the set
+            runtime.accountData[key + 'List'] = { type: 'connection', name: modelName }
+            break
+          case 'single':
+            runtime.accountData[key] = { type: 'node', name: modelName }
+            break
+          default:
```

Comment:
We have `UnreachableCaseError` to have exhaustiveness check.

Reply:
Yes I might add this at some point but I'd rather avoid importing more dependencies/APIs from Ceramic if that can be avoided.